### PR TITLE
System in place for different popups

### PR DESCRIPTION
**Pull request by VeeNinetyFour**

### CODE REVIEW

**Review by Mohamed1651 — APPROVED** [review]
Error proof should work!

**Review by LucasOudhuis — APPROVED** [review]
Updating misc.xml is not necessary

**Review by avgog — DISMISSED** [review]
The buttons of CMGT and Communication on the Open day details doesn't work.
That needs to be fixed.

**Comment by VeeNinetyFour** [comment]
> The buttons of CMGT and Communication on the Open day details doesn't work.
> That needs to be fixed.

mate you probably forgot to pull 😂 works for me

File: app/src/main/java/com/group5/cmiopenday/openDayDetails.java [hunk 1]
```diff
@@ -58,11 +58,14 @@ public void onClick(View v) {
                 Toast.makeText(openDayDetails.this, "test1", Toast.LENGTH_SHORT).show();
```
Comment (LucasOudhuis) [inline app/src/main/java/com/group5/cmiopenday/openDayDetails.java:58]: Change this svp

**Review by Dinesh0966028 — APPROVED** [review]
I see that you added some new activities to the manefest. And also an few buttons.
Also you changed the poppup activity. I really like the how coded with less Activities. We only need to change the strings, but that's oke for now.

**Pull request merged**